packages/data/schema/system/src/model/system-settings/inter_agency/interagency.schema.ts

```
@@ -0,0 +1,52 @@
+// @ts-ignore
+import { Schema, model } from 'mongoose';
+import { IInteragencyConfig } from './interface';
+import { ICreatedByAdmin } from '@isyss-cdm/interface';
+
+export interface IInteragencyConfig extends Document {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'IInteragencyConfig'. Consider to delete it or rename it.

'IInteragencyConfig' is defined here:


(lint/suspicious/noRedeclare)